boostrap_test.go

```diff
@@ -1,24 +1,24 @@
 package go_datadog_lib
 
 import (
-	"testing"
+    "testing"
 
-	"github.com/coopnorge/go-datadog-lib/config"
+    "github.com/coopnorge/go-datadog-lib/config"
 )
 
 func TestDatadog(t *testing.T) {
-	ddCfg := config.DatadogConfig{}
+    ddCfg := config.DatadogConfig{}
 
-	StartDatadog(ddCfg, false)
+    StartDatadog(ddCfg, false, false)
 
-	ddCfg = config.DatadogConfig{
-		Env:            "local",
-		Service:        "Test-Go-Datadog-lib",
-		ServiceVersion: "na",
-		DSD:            "unix:///tmp/",
-		APM:            "/tmp",
-	}
+    ddCfg = config.DatadogConfig{
+        Env:            "local",
+        Service:        "Test-Go-Datadog-lib",
+        ServiceVersion: "na",
+        DSD:            "unix:///tmp/",
+        APM:            "/tmp",
+    }
 
-	StartDatadog(ddCfg, true)
-	GracefulDatadogShutdown()
+    StartDatadog(ddCfg, true, true)
+    GracefulDatadogShutdown()
```

Comment:
Here there are a mix of whitespace changes and actual changes

Reply:
Listen do you have constructive feedback or just looking for what to comment?